### DISCUSSION TITLE
Categories filtering invisible to user

### DISCUSSION THREAD

**Discussion by tim-at-topos** [quarto-dev/quarto-cli#11328] [Feature Requests]
I'm using a listing for blog posts on my blog, and I have categories enabled because they're a great way of tagging blog posts. However, I'm running into some user feedback issues that I don't know how to solve.

The main problem is that when a user clicks a category (in the `numbered`, `unnumbered`, or `cloud` lists of categories) on the main listing page, and the blog posts are then filtered to only show those with the corresponding category, then

1. There isn't any text that says "you are now viewing only posts tagged with X" to make clear to the user that they're no longer viewing all posts (or, for example, that the search bar will now only search amongst the currently filtered posts).
2. If I use any setting other than `categories: true` (e.g. `cloud`, which I prefer), then there isn't an "All categories" option, and clicking again on the selected category doesn't disable the filtering.

I can half solve problem (1) by using some CSS to make the currently selected category underlined or something, but it's still not very clear. I think problem (2) is maybe more of a bug report?

I'd be really happy with *any* of the following solutions, if they're possible! But I'll list them in order of preference anyway...

1. Completely disable all clicking capability for the categories, but still retain the ability to filter them through the search bar
2. Dynamically add text somewhere in or around the listing saying "you are currently only viewing posts tagged with X"

**Comment by mcanouil** [comment]
You can set simple (S)CSS rules for that with `:hover` and/or `active` etc.

- https://github.com/mcanouil/quarto-extensions/blob/main/assets/stylesheets/theme.scss
- https://m.canouil.dev/quarto-extensions/

PS: prefer the use of the GitHub Discussion board for discussion/question/feature requests and keep the issues for bugs.

**Reply by tim-at-topos** [reply]
I think that problem (2) is actually a bug — should I submit that as an issue?

**Comment by mcanouil** [comment]
Regarding your other requests, I believe those are more involving and are likely to be done in JavaScript.
For instance, you could write a JavaScript script to check the selected value and add text wherever you want (as a modal?) or add tooltip.
